README.md

```diff
@@ -8,16 +51,238 @@ so don't try to use it for anything close to a production environment.
+      cluster_version: .status.clusterClaims[]| select(.name == "version.openshift.io").value,
+      cluster_topology: .status.clusterClaims[]|select(.name =="controlplanetopology.openshift.io").value}}'
+    - |
+      .metadata.labels["name"] as $name |
+      {
+        name: $name,
+        alias: $name
+      }
+  metadataServerConfig:
+    enabled: true
+  resourceServerConfig:
+    # A route must be created in the open-cluster-management namespace (RHACM) to expose the search collector Pod
+    backendURL: https://search-api-open-cluster-management.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com
+    enabled: true
+```
+
+Apply the patch to the hub cluster:
+
+```bash
+$ oc apply -f inventory_sample.yaml 
+inventory.o2ims.oran.openshift.io/default configured
+```
+The following Pods are now running on the oran-o2ims namespace:
+
+```sh
+$ oc get pods
+NAME                                             READY   STATUS    RESTARTS   AGE
+deployment-manager-server-687874c4dc-mbvvf       2/2     Running   0          18s
+metadata-server-7bb59dc878-b8gvz                 2/2     Running   0          18s
+oran-o2ims-controller-manager-785dcdd5fb-b4l5w   2/2     Running   0          88s
+resource-server-5bd78fd77d-mf6xh                 2/2     Running   0          18s
+```
+
+Several routes were created in the same namespace too. The `HOST` column is the URI where the o2ims API will be listening from outside the OpenShift cluster, for instance where the SMO will connect to.
+
+```sh
+$ oc get route -n oran-o2ims
+NAME        HOST/PORT                                                   PATH                                                   SERVICES                    PORT   TERMINATION          WILDCARD
+api-2fv99   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /                                                      metadata-server             api    reencrypt/Redirect   None
+api-cbvbz   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/resourceTypes        resource-server             api    reencrypt/Redirect   None
+api-v92jb   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/deploymentManagers   deployment-manager-server   api    reencrypt/Redirect   None
+api-xllml   o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /o2ims-infrastructureInventory/v1/resourcePools        resource-server             api    reencrypt/Redirect   None
+```
+
+Now, you can check the status of the different components of the O2IMS operator by examining the `status` field of the inventory CR.
+
+```sh
+$ oc get inventory default -ojson | jq .status.deploymentStatus.conditions
+```
+```json
+[
+  {
+    "lastTransitionTime": "2024-11-14T11:29:02Z",
+    "message": "Registered with SMO at: http://10.6.85.1:1080",
+    "reason": "SmoRegistrationSuccessful",
+    "status": "True",
```

Comment:
This condition will only be true if the example above includes the SMO attributes... but we should edit it to replace the IP address and port with something like `smo.example.com`